cpp/perspective/CMakeLists.txt

```diff
@@ -548,7 +564,9 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
             set(module_install_rpath "@loader_path/")
 	    else()
             set(module_install_rpath "\$ORIGIN")
-	    endif()
+		endif()
+	else()
+		set(CMAKE_SHARED_LIBRARY_PREFIX lib)
```

Comment:
@sc1f we should consider not prefixing with lib, and setting the prefixes to "" (this will mean we need to change python imports from `libbinding` to just `binding`, or preferable `_binding`)